main.go

```diff
@@ -16,6 +16,10 @@ import (
 	"github.com/bitrise-io/go-utils/log"
 )
 
+const (
+	stepID = "steps-cache-pull"
```

Comment:
stepID should be the step's ID I think: `cache-pull`
https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/cache-pull